MAUI/DataForm/Validation.md

```diff
@@ -0,0 +1,236 @@
+
+{% tabs %}
+{% highlight c# %}
+
+    private string name;
+
+    [Required(AllowEmptyStrings = false, ErrorMessage = "Name should not be empty")]
+    [StringLength(10, ErrorMessage = "Name should not exceed 15 characters")]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Date range attribute
+
+You can validate the date time value using date range attribute.
+
+{% tabs %}
+{% highlight c# %}
+
+    private DateTime joinDate;
+
+    [DataType(DataType.Date)]
+    [DataFormDateRange(MinimumDate = "01/01/2022", MaximumDate = "31/12/2022", ErrorMessage = "Join date is invalid")]
+    public DateTime JoinDate { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Validation mode
+
+The [ValidationMode] determines when the value should be validated.
+
+The supported validation modes are as follows:
+
+* LostFocus
+* PropertyChanged
+* Manual
+
+{% tabs %}
+{% highlight xaml %}
+
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="ValidationMode.MainPage">
+
+    <DataForm:SfDataForm
+        x:Name="DataForm" 
+        ValidationMode="LostFocus">
+    </DataForm:SfDataForm>
+
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.DataForm.ValidationMode = DataFormValidationMode.LostFocus;
+
+{% endhighlight %}
+{% endtabs %}
+
+### LostFocus
+
+If the commit mode is LostFocus, the value will be validated when the editor lost its focus.
+
+### PropertyChanged
+
+If the commit mode is PropertyChanged, The value will be validated immediately when it is changed.
+
+### Explicit
+
+The value should be validated manually by calling the [SfDataForm.Validate] or [SfDataForm.Validate(new List<string>())] method.
+
+The following code validates the value of all the properties in the data object:
+
+{% tabs %}
+{% highlight c# %}
+
+this.DataForm.Validate();
+
+{% endhighlight %}
+{% endtabs %}
+
+To validate the specific list of property value, pass the property names as argument.
+
+{% tabs %}
+{% highlight c# %}
+this.DataForm.Validate(new List<string>() {"FirstName", "Age" });
+{% endhighlight %}
+{% endtabs %}
+
+You can determine whether the data form or property is valid or not by using the `Validate` method.
+
+{% tabs %}
+{% highlight c# %}
+
+        bool isValid = DataForm.Validate();
+        bool isPropertyValid = DataForm.Validate(new List<string>());
+
+{% endhighlight %}
+{% endtabs %}
+
+If the data form or property is valid, `true` will be returned. Or else `false` will be returned.
+
+N> For validating value, the new value should be committed in data object. So, `DataFormValidationMode` takes higher priority than `DataFormCommitMode`.
+
+### Custom validation through events
```

Comment:
Custom validation through events => topic totally wrong.

It should be like Get Validation details through events. Change codes as well

Reply:
changes done